SnackOverflow/src/main/java/ro/utcn/sd/vasi/SnackOverflow/repository/jdbc/JdbcGeneralRepository.java

```diff
@@ -0,0 +1,141 @@
+package ro.utcn.sd.vasi.SnackOverflow.repository.jdbc;
+
+import com.sun.rowset.internal.Row;
+import lombok.RequiredArgsConstructor;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.jdbc.core.simple.SimpleJdbcInsert;
+import ro.utcn.sd.vasi.SnackOverflow.model.HasIntId;
+import ro.utcn.sd.vasi.SnackOverflow.repository.api.GeneralRepository;
+
+import javax.persistence.*;
+import java.lang.reflect.Field;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.time.ZonedDateTime;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+//is it wrong to use same rowmapper?
```

Comment:
No. RowMapper is essentially a pure function (no side effects, no state); you can reuse the same instance as much as you want.